docs/ConnectWallet.md

```diff
@@ -1,30 +1,61 @@
+### Use an existing hardware wallet
 
-### Internal BTCPay Wallet
+Hardware wallets provide a good balance between security and ease of use. If you already have a hardware wallet set up, you can easily use it with your BTCPay Server. Thanks to built-in [hardware wallet integration](Vault.md) the xpub key from the hardware wallet is automatically added to your BTCPay Server. The integration further allows you to spend funds received to your store within BTCPay's [internal wallet](Wallet.md)
 
-- [Ledger Wallet](LedgerWallet.md)
-- [Coldcard Wallet]( 	ColdCardWallet.md)
+:::tip
+If you own a hardware wallet, follow the instructions on how to [use a an existing hardware wallet with your BTCPay Server](Vault.md)
+:::
 
-### External Wallets
+### Use an existing software wallet
+
+Using an existing software wallet assumes that you already have an external wallet created and backed up. In theory, any mobile/desktop wallet that provides an extended public key should work, however, most wallets have technical limitations [(gap-limit)](FAQ/FAQ-Wallet/#missing-payments-in-my-software-or-hardware-wallet) that may cause serious user-experience problems for you later on.
+
+For that reason, we recommend that you only use software wallets listed below.
 
-- [Wasabi Wallet](WasabiWallet.md)
 - [Electrum Wallet](ElectrumWallet.md)
+- [Wasabi Wallet](WasabiWallet.md)
+
+Click on the link above you will be redirected to a step by step tutorial on how to set up particular software wallet with BTCPay Server.
+
+To spend and manage the funds received to your external software wallet, you can use an [internal BTCPay Wallet](Wallet.md) and sign a transaction with your private key or simply manage the funds in that external wallet itself.
+
+## Create a new wallet
+
+If you do not have an existing wallet, you can generate it within your BTCPay Server. If are using a [third-party host](ThirdPartyHosting.md), this option needs to be explicitly enabled by the server admin. Generating a new wallet in an environment you don't trust is discouraged.
+
+:::warning
+When a new wallet is generated, BTCPay Server will show you a twelve word recovery seed. After the initial display, the seed is wiped from the server, unless a [hot wallet option](HotWallet.md) is enabled.
+
+Anyone with access to your recovery seed can also access and steal all your funds. Securely back up your seed by writing it down and keep it in a safe place. Do not photograph it or store it in a digital format.
+:::
+
+Follow the [instructions here to learn how to generate a new wallet](CreateWallet.md).
+
+## Summary
+
+There are many options on how set up a wallet with your BTCPay Server. It's up to you to decide what works best for your use-case. Once you configure a wallet, you're all set. The next step, is to [start exploring the features](WhatsNext.md).
```

Comment:
>It's up to you to decide what works best for your use-case

I'm not sure about this one. Do we want to give some kind of guide or chart maybe.. Or at the very least, when creating a wallet in btcpay might be useful or chosen over the other hardware/software? Since they likely can't find that information very easily if they search something basic like: "create a new wallet in btcpay risks/benefits/tradeoffs" etc.

Reply:
Do you have a specific example here? My idea was to provide information and let user decide based on security/ease of use. For chart do you mean like something we have for choosing deployment?